data/monster/monsters/adult_goanna.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Adult Goanna" nameDescription="an adult goanna" race="blood" experience="6650" speed="420">
+	<health now="8300" max="8300" />
+	<look type="1195" corpse="34061" />
+	<targetchange interval="2000" chance="5" />
+	<flags>
+		<flag attackable="1" />
+		<flag hostile="1" />
+		<flag summonable="0" />
+		<flag convinceable="0" />
+		<flag illusionable="0" />
+		<flag isboss="0" />
+		<flag ignorespawnblock="0" />
+		<flag pushable="0" />
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="4" />
```

Comment:
Wiki documentation read "close distance" which is confusing, its actually melee. I fixed the wiki entry.
```suggestion
		<flag targetdistance="1" />
```